src/strymon_runtime/src/coordinator/dispatch.rs

```diff
@@ -43,6 +43,14 @@ impl Dispatch {
 
                 self.handle.spawn(submission);
             }
+            "Termination" => {
```

Comment:
Could we use the associated `str` from [rpc](https://github.com/strymon-system/strymon-core/pull/15/files#diff-f1de8aa2e8d90bfbcc718d9b3e71b491R51) here?

Reply:
Yes, good catch!